deps/v8/gypfiles/toolchain.gypi

```diff
@@ -61,7 +61,7 @@
     'v8_android_log_stdout%': 0,
 
     # Force disable libstdc++ debug mode.
-    'disable_glibcxx_debug%': 0,
+    'disable_glibcxx_debug%': 1,
```

Comment:
FWIW in V8 5.6 `Release` will return a `std::pair<uint8_t*, size_t>` instead of a vector.